torch/_inductor/codegen/cpp.py

```diff
@@ -3164,15 +3164,15 @@ def is_lowp_fp_scheduler(self, scheduler_node: SchedulerNode):
                 ):
                     continue
 
-                # Fast path if all operations can support bf16/fp16 without converting to fp32
-                if _node.target not in [
-                    "load",
-                    "store",
-                    "abs",
-                    "neg",
-                    "output",
-                ]:
-                    return False
+                # # Fast path if all operations can support bf16/fp16 without converting to fp32
+                # if _node.target not in [
+                #     "load",
+                #     "store",
+                #     "abs",
+                #     "neg",
+                #     "output",
+                # ]:
+                #     return False
```

Comment:
Please note that we don't need to handle these things specially if we always do 2xfp32 for fp16/bf16.